src/screens/Evaluation/TimestampedFeedbackList.js

```diff
@@ -1,23 +1,66 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import TimestampedFeedback from './TimestampedFeedback';
+import { ScrollView } from 'react-native-gesture-handler';
+import {
+  StyleSheet, View, Text, TouchableWithoutFeedback,
+} from 'react-native';
+import Post from '../../components/Post/Post';
 
-export default function TimestampedFeedbackList({ evaluations }) {
-  let listItems = [];
-  if (typeof evaluations !== 'undefined') {
-    listItems = evaluations.map((i) => (
-      <TimestampedFeedback
-        startTime={i.startTime}
-        endTime={i.endTime}
-        comment={i.comment}
-      />
-    ));
+const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'row',
+    width: '78%',
+  },
+  text: {
+    borderRadius: 5,
+    marginTop: 10,
+    marginLeft: 10,
+    padding: 10,
+    color: '#ffffff',
+    fontWeight: 'bold',
+    fontSize: 14,
+    backgroundColor: '#2196F3',
+    height: 40,
+  },
+});
+
+export default function TimestampedFeedbackList({ evaluations, videoPlayer }) {
+  function convertToMinSec(totalSeconds) {
+    let seconds = totalSeconds % 60;
+    if (seconds < 10) {
+      seconds = `0${seconds}`;
+    }
+
+    const minutes = Math.floor(totalSeconds / 60);
+    return `${minutes}:${seconds}`;
   }
+
+  const listItems = evaluations.map((i) => (
```

Comment:
i think i would prefer to be verbosity over simplicity, so clarifying what ```i``` is like ```evaluationsListItem``` would be more descriptive. Since i could be a counter, item, etc.